docs/zero_boot/service_creator/csv_parser.py

```diff
@@ -0,0 +1,444 @@
+import csv
```

Comment:
You might need to move this to `utils/zero_boot/service_creator/`?

Reply:
indeed, the full folder should be move there actually